geode-core/src/main/java/org/apache/geode/internal/admin/remote/ShutdownAllRequest.java

```diff
@@ -285,7 +285,8 @@ public void process(DistributionMessage msg) {
     }
 
     public Set getResults() {
-      logger.debug("{} shutdownAll returning {}", this, results, new Exception("stack trace"));
+      logger.debug("{} shutdownAll returning {}", this,
+          results/* , new Exception("stack trace") */);
```

Comment:
This looks like it can be removed.

Reply:
I didn't want to remove someone else's stack trace Exceptions altogether but they were getting in the way of debugging other issues.